extension/intents/nicknames/nicknames.js

```diff
@@ -114,3 +102,43 @@ intentRunner.registerIntent({
     intentRunner.unregisterPageName(name);
   },
 });
+
+intentRunner.registerIntent({
+  name: "nicknames.pause",
+  async run(context) {
+    if (context.parentRoutine === undefined) {
+      const exc = new Error("Command not available");
+      exc.displayMessage = "Command not available";
+      throw exc;
+    }
+    context.parentRoutine.stop = true;
```

Comment:
`.stop` should be a function instead of a flag. (It might just set `this._stop = true`, but from the outside it should be a function)